src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -88,25 +87,26 @@
 
 .pf-c-empty-state__content {
   max-width: var(--pf-c-empty-state__content--MaxWidth);
-
-  > .pf-c-title.pf-m-lg {
-    font-size: var(--pf-c-empty-state__content--c-title--m-lg--FontSize);
```

Comment:
This is good, but I'm wondering if we should use the title component here, or have an empty state title element of the empty state component instead (`.pf-c-empty-state__title`).

FWIW, this override came from a design request to change the empty state's title font-size from lg to xl along with another change (I think with an opt-in to a different font). Since we're using the title component here, and the size is tied to a modifier class, an HTML change has to happen to get that change, and we try to avoid that for core users since they'll have the wrong font size until they change their HTML. HTML changes needed to fix things from a release are considered breaking - though I wouldn't necessarily say it's "breaking" to use a lg font here instead of the intended xl. But since we wanted everyone to get this font size change from lg to xl , we styled the title's large modifier to use the xl font - which is just a messy compromise. In a number of other components, we've removed use of the title component and replaced it with an element of the component, when the title's styling is tied more to the component and its variations than just to the user's preference of any sort of title. A user would still have the option of passing a `<Title>` in if they wanted to.

What I was thinking is a block for the header, body, and footer/actions, and using a consistent container class for blocks of actions.

```
empty-state
  empty-state__header (new)
    empty-state__icon
    empty-state__title (new)
      this can be any node - text, a layout, a title component, whatever
  empty-state__body
  empty-state__footer (new)
    empty-state__actions (new - at least one required)
    empty-state__actions (new - for any secondary/add'l actions)
```

wdyt?